packages/hop-node/src/cli/bondWithdrawal.ts

```diff
@@ -29,23 +41,36 @@ async function main (source: any) {
     throw new Error('transfer ID is required')
   }
 
-  const watcher = await getBondWithdrawalWatcher({ token, dryMode })
+  const watcher = await getBondWithdrawalWatcher({ chain, token, dryMode })
   if (!watcher) {
     throw new Error('watcher not found')
   }
 
   for (const transferId of transferIds) {
     try {
-      const dbTransfer: any = await watcher.db.transfers.getByTransferId(transferId)
+      const dbTransfer: Transfer = await watcher.db.transfers.getByTransferId(transferId)
       if (!dbTransfer) {
         throw new Error('TransferId does not exist in the DB')
       }
-      dbTransfer.attemptSwap = watcher.bridge.shouldAttemptSwapDuringBondWithdrawal(dbTransfer.amountOutMin, dbTransfer.deadline)
-      if (dbTransfer.attemptSwap && dbTransfer.destinationChainId === 1) {
+      const attemptSwap = watcher.bridge.shouldAttemptSwapDuringBondWithdrawal(dbTransfer.amountOutMin, dbTransfer.deadline)
+      if (attemptSwap && dbTransfer.destinationChainId === 1) {
```

Comment:
suggestion: also check for testnet L1 here

```suggestion
     if (attemptSwap && isL1ChainId(dbTransfer.destinationChainId)) {
```